Makefile-tests.am

```diff
@@ -109,16 +109,29 @@ vmsync:
 
 vmoverlay:
 	@if [ -z "$(SKIP_VMOVERLAY)" ]; then \
-	  env $(BASE_TESTS_ENVIRONMENT) ./tests/vmcheck/overlay.sh; \
+	  if [ -n "$(HOSTS)" ]; then \
+	    echo -n "$(HOSTS)" | xargs -P 0 -n 1 -d ' ' -I {} \
+	      env $(BASE_TESTS_ENVIRONMENT) VM={} PYTHONUNBUFFERED=1 \
+	        ./tests/vmcheck/overlay.sh; \
+	  else \
+	    env $(BASE_TESTS_ENVIRONMENT) ./tests/vmcheck/overlay.sh; \
+	  fi \
 	fi
 
 vmshell: vmsync
 	ssh -F ssh-config vmcheck
 
 # set up test environment to somewhat resemble uninstalled tests
 vmcheck: vmoverlay tests/common/compose/yum/repo/repodata/repomd.xml
-	@env VMTESTS=1 $(BASE_TESTS_ENVIRONMENT) \
-	   sh tests/vmcheck/test.sh
+	@if [ -z "$(HOSTS)" ]; then \
+	  echo "running test.sh"; \
+	  env VMTESTS=1 $(BASE_TESTS_ENVIRONMENT) \
+	    sh tests/vmcheck/test.sh; \
+	else \
+	  echo "running multitest.py"; \
+	  env VMTESTS=1 $(BASE_TESTS_ENVIRONMENT) \
+	    tests/vmcheck/multitest.py $(HOSTS); \
```

Comment:
Is there a reason not to use `multitest.py` even with 1 host?  That way we're testing it more...